user-management/script.js

```diff
@@ -185,4 +186,13 @@ async function showUserDataList(page) {
   generateUserList(userData, true);
 }
 
-showUserDataList(page);
+async function init() {
+  const isSuperUser = await checkUserIsSuperUser();
+  if (isSuperUser) {
+    showUserDataList(page);
```

Comment:
where is this method?

Reply:
> where is this method?

Are you talking about the `showUserDataList()` method?

Reply:
yes

Reply:
Its here https://github.com/Real-Dev-Squad/website-dashboard/blob/develop/user-management/script.js#L183